.eslintrc.js

```diff
@@ -1,15 +1,96 @@
 module.exports = {
-  env: {
-    es2021: true,
-    node: true
-  },
   extends: [
-    'standard'
+    'plugin:node/recommended',
+    'plugin:prettier/recommended',
+    'eslint-config-airbnb-base',
+    'plugin:eslint-comments/recommended',
+    'plugin:promise/recommended',
+    // 'plugin:unicorn/recommended', TODO: wait for esm support
+    'standard',
+    'prettier',
   ],
+  plugins: ['standard', 'eslint-comments', 'promise', 'unicorn'],
+  parser: '@babel/eslint-parser',
```

Comment:
Use the babel parser to support full set of new node experimental features since eslint only supports the latest final ECMAScript standard and changes to core rules often lag behind

Reply:
No more **eslint-errors** ✨ I vote for it 🙌